File: src/app/Model/UserLogin.ts
export class UserLogin {
  constructor() {
  }
  public id;
  public name: string;
  public password: string;
  public ConnectionID: string;
  public isConnect: string;

}
export class Messages {
  constructor() {}
  public id;
  public sender: string;
  public recevier: string;
  public message: string;
  public time: Date;
}

File: Models/UserLogin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using static ChatApplication.Data.ChatDBContext;

namespace ChatApplication.Models
{
    public class UserLogin
    {
    public int id { get; set; }
    public string name { get; set; }
    public string password { get; set; }
    public string ConnectionID { get; set; }
    public string isConnect { get; set; }

  }
  public class Messages
  {
    public int id { get; set; }
    public string sender { get; set; }
    public string recevier { get; set; }
    public string message { get; set; }
    public DateTime time { get; set; }
  }
}
